pkg/manifests/manifests.go

```diff
@@ -0,0 +1,193 @@
+package manifests
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/suse-edge/edge-image-builder/pkg/fileio"
+	"github.com/suse-edge/edge-image-builder/pkg/http"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+	"go.uber.org/zap"
+	"gopkg.in/yaml.v3"
+)
+
+func configureManifests(ctx *image.Context) error {
+	var downloadedManifests []string
+	var combinedManifestPaths []string
+	var extractedImages []string
+	var err error // created here to avoid scoping issues with "downloadedManifests" when using := on line 29
+
+	if len(ctx.ImageDefinition.Kubernetes.Manifests.URLs) != 0 {
+		downloadDestination := filepath.Join(ctx.CombustionDir, "downloaded-manifests")
```

Comment:
Just FYI, if I have the time I'd like to introduce a better structure of the Combustion directory where everything related to Kubernetes falls under `/combustion/kubernetes`. Until then, this is fine.